content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,103 @@
+
+## Usage
+
+```
+library(epidatr)
+# Obtain the smoothed covid-like illness (CLI) signal from the Facebook survey as it was on April 10, 2021 for the US
+epidata <- pub_covidcast(
+    source = "fb-survey",
+    signals = "smoothed_cli",
+    geo_type = "nation",
+    time_type = "day",
+    geo_values = "us",
+    time_values = epirange(20210101, 20210601),
+    as_of = 20210601
+)
+epidata
+```
+
+```
+# A tibble: 151 × 15
+   geo_value signal       source geo_type time_type time_value
+   <chr>     <chr>        <chr>  <fct>    <fct>     <date>
+ 1 us        smoothed_cli fb-su… nation   day       2021-01-01
+ 2 us        smoothed_cli fb-su… nation   day       2021-01-02
+ 3 us        smoothed_cli fb-su… nation   day       2021-01-03
+ 4 us        smoothed_cli fb-su… nation   day       2021-01-04
+ 5 us        smoothed_cli fb-su… nation   day       2021-01-05
+ 6 us        smoothed_cli fb-su… nation   day       2021-01-06
+ 7 us        smoothed_cli fb-su… nation   day       2021-01-07
+ 8 us        smoothed_cli fb-su… nation   day       2021-01-08
+ 9 us        smoothed_cli fb-su… nation   day       2021-01-09
+10 us        smoothed_cli fb-su… nation   day       2021-01-10
+# ℹ 141 more rows
+# ℹ 9 more variables: direction <dbl>, issue <date>,
+#   lag <dbl>, missing_value <dbl>, missing_stderr <dbl>,
+#   missing_sample_size <dbl>, value <dbl>, stderr <dbl>,
+#   sample_size <dbl>
+# ℹ Use `print(n = ...)` to see more rows
+```
+
+## Installation
+
+You can install the stable version of this package from CRAN:
+
+```
+install.packages("epidatr")
+pak::pkg_install("epidatr")
+renv::install("epidatr")
+```
+
+Or if you want the development version, install from GitHub:
+
+```
+# Install the dev version using `pak` or `remotes`
+pak::pkg_install("cmu-delphi/epidatr")
+remotes::install_github("cmu-delphi/epidatr")
+renv::install("cmu-delphi/epidatr")
+```
+
+### API Keys
+
+The Delphi API requires a (free) API key for full functionality. To generate
+your key, register for a pseudo-anonymous account
+[here](https://api.delphi.cmu.edu/epidata/admin/registration_form) and see more
+discussion on the [general API
+website](https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html). The
+`epidatr` client will automatically look for this key in the R option
+`delphi.epidata.key` or in the environment variable
+`DELPHI_EPIDATA_KEY`. We recommend storing your key in `.Renviron` file, which R
```

Comment:
```suggestion
`DELPHI_EPIDATA_KEY`. We recommend storing your key in a `.Renviron` file, which R
```

(or go further and recommend `~/.Renviron`?)

Reply:
Good catch, this definitely needs an article!  I'm not sure if "a" or "an" is appropriate here -- i would use the former if pronouncing the "dot" and the latter if leaving it out...  Maybe we should use "the" or "your"?